firmware/src/can/can_endpoints.c

```diff
@@ -399,7 +399,7 @@ uint8_t CAN_SetIntegratorGains(uint8_t buffer[], uint8_t *buffer_len, bool rtr)
 
 uint8_t CAN_DeviceInfo(uint8_t buffer[], uint8_t *buffer_len, bool rtr)
 {
-    const uint32_t idr = PAC55XX_INFO2->PACIDR;
+    const uint32_t idr = PAC55XX_INFO1->UNIQUEID[0];
```

Comment:
We could return a xor of all uniqueid bits like so: `PAC55XX_INFO1->UNIQUEID[0] ^ PAC55XX_INFO1->UNIQUEID[1] ^ PAC55XX_INFO1->UNIQUEID[2]` , what do you think?

Reply:
I'm not sure this would return assure a unique ID back as from reading the other components of `PAC55XX_INFO1->UNIQUEID` are not necessarily unique per-device.

Looking at a general EPC implementation, it often functions as [here](https://www.tutorialspoint.com/electronic-product-code-epc) (and from testing between two tinymovr boards I saw this replicated, where only the value in `UNIQUEID[0]` was different).

Reply:
I'm not sure whether Qorvo implements EPC in this code. [ST has their own numbering scheme](https://electronics.stackexchange.com/questions/338559/stm32-uid-decoding) for instance. Nonetheless, I'm going to merge this because it is certainly useful. I'll also send a mail to Qorvo for further information.

Thanks!